gapis/perfetto/android/trace.go

```diff
@@ -291,12 +291,16 @@ func (p *Process) captureWithClientApi(ctx context.Context, start task.Signal, s
 			return 0, log.Err(ctx, nil, "Timed out in waiting for trace session ready")
 		}
 	}
+
+	// TODO Find a way to reliably know when Perfetto/producers are ready (b/147388497)
```

Comment:
Nit: put the bug id right next to the TODO: `TODO(b/147388497): Find a ...`

See: go/tott/579

Reply:
Thanks for the reference, done in a new commit!